src/ui/components/ConfirmButton/index.js

```diff
@@ -18,6 +18,7 @@ type Props = {|
   className?: string,
   confirmButtonText?: string | null,
   confirmButtonType?: string,
+  id: string,
```

Comment:
Because we don't have 100% Flow coverage, this also needs to be protected in the `render()` function like:

```js
render() {
  invariant(this.props.id, 'The id property is required');
}
```

Reply:
Actually, a few follow up ?s here: So should I do the same with the message prop (and other required props)? The message text is actually used here in the render vs the id which is only used in the extract function).